build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2024.2.1"
+    id "edu.wpi.first.GradleRIO" version "2024.3.1"
```

Comment:
can we check units one this?

Reply:
yeah so actually you can see it says "version" right there and it's in units of versions so it's correct I would say

Reply:
ok cool approved